vuepress/docs/v7.0/tutorials/devops/invoking-api.md

```diff
@@ -1,186 +1,94 @@
-# Tutorial: Invoking Entando Core APIs
+---
+sidebarDepth: 2
+---
 
-## General Overview
+# Invoking Entando Core APIs
 
-Entando supports Swagger and it is accessible, if activated, from its
-web interface from the base application URL at /api/swagger-ui.html. A
-complete list of all available core APIs can be obtained from Swagger.
+Entando Core APIs can be called directly with Swagger or tools like curl and Postman. Entando uses Swagger to automatically generate OpenAPI documentation that allow such interactions. This tutorial describes how to set up an environment ready to invoke the APIs using Postman.
 
-Core APIs can be invoked directly from Swagger or from any other
-suitable tool, i.e., curl or Postman are two common ones. Postman has
-been extensively used in API testing and automated API testing because
-of the possibilities it offers and it is the suggested tool.
+All Entando core APIs are accessible from a base URL such as localhost:8080/entando-de-app/api. For example, the page controller is available under /pages at the URL localhost:8080/entando-de-app/api/pages. The page template controller is available at localhost:8080/entando-de-app/api/pageModels.
 
-All Entando core APIs are accessible from the base URL, for example:
-localhost:8080/entando-de-app/api. For example, the Page Controller will
-be available under /pages from the Base URL (i.e.
-localhost:8080/entando-de-app/api/pages), or the Page Template Controller
-will available under /pageModels (i.e.
-localhost:8080/entando-de-app/api/pageModels) and so on.
+To perform extensive testing with APIs on Postman, set up an environment and define a variable to host the access token saved with a POST request to /oauth/token. This should provide access to all API endpoints and actions for the user's defined privileges. 
```

Comment:
creating an environment is a literal thing in Postman. You create and send Post requests, and the saving process is done through the variable so it can't be saved by just issuing a post request.